src/ui/components/Rating/index.js

```diff
@@ -89,8 +88,9 @@ export class RatingBase extends React.Component {
     }
     let description;
     if (rating) {
+      const roundedRating = Math.round(rating * 2) / 2;
```

Comment:
oh, right, `Math.round()` doesn't let you specify precision. The division hack is confusing. 

How about this instead?
````js
{ rating: i18n.formatNumber(rating.toFixed(1)) }
````